### DISCUSSION TITLE
MBC Sports+ [KR]

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#368] [Show and tell]
### Channel ID

MBCSportsPlus.kr

### Stream URL

http://58.142.60.62:9981/stream/channelid/509330915?profile=pass

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://58.142.60.62:9981/stream/channelid/509330915?profile=pass (offline)
<img width="473" alt="image" src="https://user-images.githubusercontent.com/7253922/232247934-2dbb3256-9b09-4d49-9b0a-4d864171934f.png">
